test/powershell/engine/Help/HelpSystem.Tests.ps1

```diff
@@ -469,3 +469,20 @@ Describe 'help can be found for AllUsers Scope' -Tags @('Feature', 'RequireAdmin
         $helpObj.description | Out-String | Should -Match $CmdletName
     }
 }
+
+Describe 'help renders when using a PAGER with a space in the path' -Tags 'CI' {
+    BeforeAll {
+        $SavedEnvPager = $env:PAGER
+        $env:PAGER = "C:\Program Files\Git\usr\bin\less.exe"
```

Comment:
@JamesWTruher Any suggestions on how we could avoid using less.exe? 
@pougetat We do not expect that git must be installed on test environments.

Reply:
Hey @adityapatwardhan , @JamesWTruher 
I made the modifs that were asked. Any update on this particular question ? :)

Reply:
For testing, we don't need to use an actual pager.  If the test is to validate piping data to something that has a space in the path, why not just use something like:

Join-Path $TestDrive "folder with spaces\pager.ps1"

Where pager.ps1 is simply takes input from pipeline (some test text) and outputs a hash you can verify?